opencti-platform/opencti-front/src/private/components/common/lists/FilterAutocomplete.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { Dispatch, FunctionComponent, SyntheticEvent, useState } from 'react';
+import TextField from '@mui/material/TextField';
+import * as R from 'ramda';
+import Autocomplete from '@mui/material/Autocomplete';
+import makeStyles from '@mui/styles/makeStyles';
+import { AutocompleteValue } from '@mui/material';
+import ItemIcon from '../../../../components/ItemIcon';
+import { useFormatter } from '../../../../components/i18n';
+import useSearchEntities from '../../../../utils/filters/useSearchEntities';
+import { Theme } from '../../../../components/Theme';
+import SearchScopeElement from './SearchScopeElement';
+import { onlyGroupOrganization } from '../../../../utils/filters/filtersUtils';
+import { HandleAddFilter } from '../../../../utils/hooks/useLocalStorage';
+
+const useStyles = makeStyles<Theme>((theme) => ({
+  icon: {
+    paddingTop: 4,
+    display: 'inline-block',
+    color: theme.palette.primary.main,
+  },
+  text: {
+    display: 'inline-block',
+    flexGrow: 1,
+    marginLeft: 10,
+  },
+}));
+
+interface OptionValue {
```

Comment:
That is already declared somewhere as Option

Reply:
In ReferenceField.tsx there are this entity :
export interface Option {
  value: string
  label: string
  color?: string
  [key: string]: ReactNode
}

should I extend it ?

Reply:
I think I will add the missing property to this existing entity. Like that, we have the same entity for the autocomplete field on both side.